hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/bucket/CreateBucketHandler.java

```diff
@@ -53,11 +53,11 @@
 
   // TODO: LEGACY should be removed and should not be exposed to the end user
   // we will revisit during the client side defaulting behaviour
-  enum AllowedBucketLayouts { FILE_SYSTEM_OPTIMIZED, OBJECT_STORE, LEGACY }
+  enum AllowedBucketLayouts { FILE_SYSTEM_OPTIMIZED, OBJECT_STORE }
```

Comment:
Please add a test case to verify the client side allowed layout validation. One idea is to add test case in TestOzoneShellHA.java. 

Sample test case snippet:

    String [] args1 = new String[]{
        "bucket", "create", "o3://" + omServiceId +
        "/volume4" + "/buckettest1", "--type",
        "LEGACY"};
    try {
      execute(ozoneShell, args1);
      Assert.fail("Should throw exception on unsupported bucket layouts!");
    } catch (Exception e) {
      GenericTestUtils.assertExceptionContains("expected one of [FILE_SYSTEM_OPTIMIZED, OBJECT_STORE] ", e);
    }

Reply:
Thank you for the help, I'm adding this test to the suite.